Assets/AnkrDemo/Scripts/HeaderWalletButton.cs

```diff
@@ -0,0 +1,57 @@
+using System;
+using AnkrSDK.Data;
+using AnkrSDK.Examples.UseCases.WebGlLogin;
+using AnkrSDK.WebGL.DTO;
+using UnityEngine;
+using UnityEngine.UI;
+
+namespace Demo.Scripts
+{
+	public class HeaderWalletButton : MonoBehaviour
+	{
+		public WalletItem WalletItem;
+		
+		[SerializeField]
+		private Image _logoContainer;
+		
+		[SerializeField]
+		private GameObject _markerContainer;
+		
+		[SerializeField]
+		private Button _button;
+		
+		public Action<Wallet> OnClickHandler;
```

Comment:
no public fields allowed. If it is an event it should be marked with event keyword. IF it is an action that you want to store from the outside use property/method.
I suppose it should be public event Action<Wallet> WalletClicked;